hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/protocolPB/OzoneManagerRequestHandler.java

```diff
@@ -386,12 +386,13 @@ private LookupKeyResponse lookupKey(LookupKeyRequest request,
         .setVolumeName(keyArgs.getVolumeName())
         .setBucketName(keyArgs.getBucketName())
         .setKeyName(keyArgs.getKeyName())
-        .setRefreshPipeline(true)
-        .setSortDatanodesInPipeline(keyArgs.getSortDatanodes())
```

Comment:
Why is "sort datanodes" flag no longer set?

Reply:
It was a mistake, thanks for catching